src/resources/views/inc/form_save_buttons.blade.php

```diff
@@ -1,26 +1,30 @@
 <div id="saveActions" class="form-group">
+    @if(!empty($saveAction))
+        <input type="hidden" name="save_action" value="{{ $saveAction['active']['value'] }}">
 
-    <input type="hidden" name="save_action" value="{{ $saveAction['active']['value'] }}">
+        <div class="btn-group">
 
-    <div class="btn-group">
+            <button type="submit" class="btn btn-success">
+                <span class="fa fa-save" role="presentation" aria-hidden="true"></span> &nbsp;
+                <span data-value="{{ $saveAction['active']['value'] }}">{{ $saveAction['active']['label'] }}</span>
+            </button>
 
-        <button type="submit" class="btn btn-success">
-            <span class="fa fa-save" role="presentation" aria-hidden="true"></span> &nbsp;
-            <span data-value="{{ $saveAction['active']['value'] }}">{{ $saveAction['active']['label'] }}</span>
-        </button>
+            @if(!empty($saveAction['options']) && is_array($saveAction['options']))
+                <button type="button" class="btn btn-success dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aira-expanded="false">
+                    <span class="caret"></span>
+                    <span class="sr-only">&#x25BC;</span>
+                </button>
 
-        <button type="button" class="btn btn-success dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aira-expanded="false">
-            <span class="caret"></span>
-            <span class="sr-only">&#x25BC;</span>
-        </button>
+                <ul class="dropdown-menu">
+                    @foreach( $saveAction['options'] as $value => $label)
+                    <li><a href="javascript:void(0);" data-value="{{ $value }}">{{ $label }}</a></li>
+                    @endforeach
+                </ul>
+            @endif
 
-        <ul class="dropdown-menu">
-            @foreach( $saveAction['options'] as $value => $label)
-            <li><a href="javascript:void(0);" data-value="{{ $value }}">{{ $label }}</a></li>
-            @endforeach
-        </ul>
-
-    </div>
-
-    <a href="{{ url($crud->route) }}" class="btn btn-default"><span class="fa fa-ban"></span> &nbsp;{{ trans('backpack::crud.cancel') }}</a>
-</div>
\ No newline at end of file
+        </div>
+    @endif
+    @if(empty($removeCancelAction))
```

Comment:
I don't see ```$removeCancelAction``` anywhere else in CRUD's codebase. How's this supposed to work?

Reply:
@tabacitu what about this ?? is it frustrating if user didn't see cancel button ?? Same reason i want to hide cancel button then how can i do this ??